05-EventAggregator/src/PrismSample.Android/MainActivity.cs

```diff
@@ -1,22 +1,26 @@
 ﻿using Android.App;
 using Android.Content.PM;
 using Android.OS;
+using Android.Widget;
+using Prism;
+using Prism.Events;
+using Prism.Ioc;
+using PrismSample.Models;
 
 namespace PrismSample.Droid
 {
-    [Activity(Theme = "@style/MainTheme",
-              ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
+    [Activity(Theme = "@style/MainTheme", ConfigurationChanges = ConfigChanges.ScreenSize | ConfigChanges.Orientation)]
     public class MainActivity : global::Xamarin.Forms.Platform.Android.FormsAppCompatActivity
     {
-        protected override void OnCreate(Bundle savedInstanceState)
+        protected override void OnCreate(Bundle bundle)
         {
             TabLayoutResource = Resource.Layout.Tabbar;
             ToolbarResource = Resource.Layout.Toolbar;
 
-            base.OnCreate(savedInstanceState);
+            base.OnCreate(bundle);
 
-            global::Xamarin.Forms.Forms.Init(this, savedInstanceState);
-            var application = new App();
+            global::Xamarin.Forms.Forms.Init(this, bundle);
+            var application = new App(new AndroidInitializer());
```

Comment:
This sample is not meant to demonstrate anything in regards to the IPlatformInitializer which is why it was explicitly excluded.